examples/ESA/ESA_WorldCereal_2021_MODELS_v100_preview.js

```diff
@@ -0,0 +1,59 @@
+// var dataset = ee.ImageCollection('ESA/WorldCereal/2021/MODELS/v100')
+var dataset = ee.ImageCollection('projects/worldcereal/assets/2021_v100')
+
+Map.setOptions('SATELLITE');
```

Comment:
Background is not saved to to the thumbnail right? The preview at the moment is kind of ugly.

Reply:
We generally fully hide the background - eg, see https://github.com/google/earthengine-catalog/blob/main/examples/ACA/ACA_reef_habitat_v2_0_preview.js

I added white background here:
https://code.earthengine.google.com/f9d015b5eb804aec4814431f68e37ae9
but now the map looks very bright, so suggestions are welcome. We can also add continent outlines using another dataset to make the map clearer.

Reply:
@simonff  I don't really know how to make a good thumbnail image given these constraints. As mentioned in earlier comment, ideally the thumbnail is a global view but I didn't get that to work. Continent outlines could help increase the clarity and we could also opt for a darker background. But I will need help getting this done because I haven't worked with thumbnails before and don't really know the constraints to create them; other comments have been addressed.

Reply:
How's this? https://code.earthengine.google.com/da7f5823dd9792c1a4d2a8cf95bb9a9a

It's pretty much impossible to fit the whole world into a square, so for global datasets we usually select some nice-looking continent-scale view (Europe, Australia, India, North America, etc)

Reply:
Looks good!